lib/govuk_web_banners/emergency_banner.rb

```diff
@@ -0,0 +1,33 @@
+require "govuk_app_config/govuk_error"
+require "redis"
+
+module GovukWebBanners
+  class EmergencyBanner
+    attr_reader :campaign_class, :heading, :short_description, :link, :link_text
+
+    def initialize(redis_client:)
+      content = content_from_redis(redis_client)
```

Comment:
As commented in the view, I think that as there's only one place that the `EmergencyBanner` class is getting the banner information, I don't think the Redis client needs to be passed in, could just be a method in this class.